workspaces/src/network/sandbox.rs

```diff
@@ -87,29 +86,35 @@ impl AllowDevAccountCreation for Sandbox {}
 
 #[async_trait]
 impl TopLevelAccountCreator for Sandbox {
-    async fn create_tla(&self, id: AccountId, sk: SecretKey) -> Result<CallExecution<Account>> {
+    async fn create_tla(
+        &self,
+        worker: Worker<dyn Network>,
+        id: AccountId,
+        sk: SecretKey,
+    ) -> Result<CallExecution<Account>> {
         let root_signer = self.root_signer()?;
```

Comment:
There's no test for this or `create_tla_and_deploy`. Definitely can be done in a separate PR but it's a bit unclear how this would be used by a developer. Would you mind adding a test for this to make sure it's usable?

My intuition is that this won't be very usable because the developer would have to do the coercion themselves. Might require a bit of re-working of this TLAC trait. Am I missing something here?

Reply:
So, there's one new thing with this PR, Networks like Sandbox/Testnet impl TLAC but worker has it's own custom impl of it but without impl the TLAC trait. So it just has the signature of `create_tla(id: AccountId, sk: SecretKey)` instead. With this, the user never really uses the TLAC trait

Reply:
Can we remove it in the next minor version then? Does it have any use? If these traits are going to be removed in the future, makes sense to do sooner than later?

Reply:
the TLAC trait is still required since we need an interface to call into the `create_tla` function for each respective network, and provides the ability to add custom networks down the line. So I don't see this trait being removed, but the `DevAccountDeployer` trait didn't make sense to stay so that one was removed